core/jazz_root-authorizer/index.js

```diff
@@ -0,0 +1,518 @@
+// =========================================================================
```

Comment:
better name will be jazz_admin-authorizer or jazz_cognito-admin-authorizer

Reply:
Created new folder "jazz_cognito-admin-authorizer" and moved all the files.